src/MultigridMesh.f90

```diff
@@ -193,9 +193,6 @@ MODULE MultigridMesh
 
   ENDTYPE InterpWeightsStructureType
 
-  !> Exception Handler for use in MatrixTypes
-  TYPE(ExceptionHandlerType),SAVE :: eMultigridMesh
```

Comment:
shouldn't we be more concerned the exception handler is never used?

Reply:
Actually the module uses the `eLinearSolverType` variable a bit.  I expected to agree with you, but turns out there is some error checking, just using a different handler.